.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+    - name: Check out
+      uses: actions/checkout@v2
+    - name: Get dependencies
+      run: go get -v -t -d ./...
```

Comment:
Do you really need this step?

Reply:
I'm kind of just messing around right now with it. Sorry for the noise. I'll add you as a reviewer on the final pass.